src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -543,6 +566,7 @@ export class InputTimePicker
           </div>
         </div>
         <calcite-popover
+          aria-label={messages.chooseTime}
           aria-live="polite"
```

Comment:
Can you remove `aria-live` here? It's also handled by the popover.

Reply:
Also `aria-label` and set `label` instead.

Reply:
Sorry, I accidentally merged it back when resolving conflicts.